app/views/Register/RegisterForm/index.tsx

```diff
@@ -1,135 +1,274 @@
-import React, { useCallback } from 'react';
-
+import React from 'react';
 import {
     Container,
+    RadioInput,
     TextInput,
     PasswordInput,
+    useInputState,
     Button,
+    useAlert,
 } from '@the-deep/deep-ui';
-
 import {
-    ObjectSchema,
     PartialForm,
-    emailCondition,
-    requiredStringCondition,
-    lengthGreaterThanCondition,
-    lengthSmallerThanCondition,
     useForm,
     getErrorObject,
     createSubmitHandler,
+    removeNull,
 } from '@togglecorp/toggle-form';
-import styles from './styles.css';
+import { useMutation, gql } from '@apollo/client';
 
-interface RegisterFields {
-    email: string;
-    fullName: string;
-    organizationName: string;
-    createPassword: string;
-    verifyPassword: string;
-    captcha?: string;
-}
+import {
+    UserUserType,
+    RegisterMutation,
+    RegisterMutationVariables,
+} from '#generated/types';
 
-type FormType = Partial<RegisterFields>;
+import {
+    transformToFormError,
+    ObjectError,
+} from '#base/utils/errorTransform';
 
-type FormSchema = ObjectSchema<PartialForm<FormType>>;
+import {
+    RegisterFormType,
+    schema,
+} from './common';
 
-type FormSchemaFields = ReturnType<FormSchema['fields']>;
+import InstitutionForm from './InstitutionForm';
+import PublisherForm from './PublisherForm';
+import SchoolForm from './SchoolForm';
+import styles from './styles.css';
 
-const schema: FormSchema = {
-    fields: (): FormSchemaFields => ({
-        email: [emailCondition, requiredStringCondition],
-        fullName: [requiredStringCondition],
-        organizationName: [requiredStringCondition],
-        createPassword: [
-            requiredStringCondition,
-            lengthGreaterThanCondition(4),
-            lengthSmallerThanCondition(129),
-        ],
-    }),
+interface UserType {
+    id: UserUserType;
+    title: string;
+}
+
+const userType: UserType[] = [
```

Comment:
We should be able to fetch these from server

Reply:
Lets leave it as is for now